src/integrationCategories/types.ts

```diff
@@ -1,8 +1,13 @@
+// Fields externalId, iconUrl, createdAt, updatedAt will be deprecated.
```

Comment:
No `externalId` in the type itself, as far as I can see

Reply:
I have returned back `externalId?: string;`